doc/deployment/deploy-to-AKS-blueegreen-using-jenkins.md

```diff
@@ -0,0 +1,200 @@
+    ```
+
+    Repeat for `todoapp-test-blue` and `todoapp-test-green`:
+    ```
+    az network public-ip update --dns-name todoapp-blue --ids /subscriptions/XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX/resourceGroups/MC_<resourcegroup>_<aks>_<location>/providers/Microsoft.Network/publicIPAddresses/kubernetes-BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB
+
+    az network public-ip update --dns-name todoapp-green --ids /subscriptions/XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX/resourceGroups/MC_<resourcegroup>_<aks>_<location>/providers/Microsoft.Network/publicIPAddresses/kubernetes-CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
+    ```
+    Note that the dns name needs to be unique in your subscription.
+
+    Now, you can visit your app by navigating to e.g.,  http://aks-todoapp.<your-location>.cloudapp.azure.com. If you see the following, you have succeeded in setting up your AKS cluster.  
+    ![Tomcat](images/tomcat.png)
+
+### Create Azure Container Registry
+
+1. Run below command to create an Azure Container Registry.
+After creation, use `login server` as Docker registry URL in the next section.
+
+   ```bash
+   az acr create -n <your-registry-name> -g <your-resource-group-name>
+   ```
+
+1. Run below command to show your Azure Container Registry credentials.
+You will use Docker registry username and password in the next section.
+
+    ```bash
+    az acr credential show -n <your-registry-name>
+    ```
+
+## Prepare Jenkins server
+1. Deploy a Jenkins Master on Azure [https://aka.ms/jenkins-on-azure]
+
+1. Connect to the server with SSH and install the build tools on the server where you will run your build:
+   
+   ```
+   sudo apt-get install git maven docker.io
+   ```
+
+1. Install additional tools needed for this example:
+
+    [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 
+
+
+    [jq](https://stedolan.github.io/jq/download/)
+    ```
+    sudo apt-get install jq
+    ```
+   
+1. Install the plugins in Jenkins. Click 'Manage Jenkins' -> 'Manage Plugins' -> 'Available', 
+then search and install the following plugins: Azure Container Service Plugin.
+
+1. Add dd a Credential in type "Microsoft Azure Service Principal" with your service principal.
+
+1. Add a Credential in type "Username with password" with your account of docker registry.
+
+## Edit the Jenkinsfile
+
+1. In your own repo, navigate to /src/aks/ and open `Jenkinsfile`
+
+2. Update:
+
+```
+    def servicePrincipalId = '<your-service-principal>'
+    def resourceGroup = '<your-resource-group-name>'
+    def aks = '<your-kubernetes-cluster-name>'
+
+    def cosmosResourceGroup = '<your-cosmodb-resource-group>'
+    def cosmosDbName = '<your-cosmodb-name>'
+    def dbName = '<your-dbname>'
+
+    def dockerRegistry = '<your-acr-name>.azurecr.io'

```

Comment:
We need to update the `dockerCredentialId` with the ID the user stored in the Jenkins instance.